packages/tooltip/test/lion-tooltip.test.js

```diff
@@ -103,6 +102,40 @@ describe('lion-tooltip', () => {
     });
   });
 
+  describe('Positioning', () => {
+    it('updates popper positioning correctly, without overriding other modifiers', async () => {
```

Comment:
Here I solved headless vs actual browser inconsistency by using a div instead of button for invoker, and also using position: absolute and then offsetting it with top, to get it to pass in both browser and headless.

Might be possible to re-enable some of the skipped test with this.. but didn't have time for it.